### ISSUE TITLE
release/index.json reliability

### ISSUE THREAD

**Issue by springmeyer** [iojs/build#94]
Yesterday https://iojs.org/download/release/index.json reported `"modules":"44"` for the `v2.0.0` release which looks correct per https://github.com/iojs/io.js/blob/v2.0.0/src/node_version.h#L48.

But today the same endpoint reports `42`. Any idea what changed? (I don't see any changes to https://github.com/iojs/build/commits/master/tools/dist/dist-indexer/ in the last day)

```json
[
{"version":"v2.0.0","date":"2015-05-04","files":["linux-armv6l","linux-armv7l","linux-x64","linux-x86","osx-x64-pkg","osx-x64-tar","src","win-x64-exe","win-x64-msi","win-x86-exe","win-x86-msi"],"npm":"2.9.0","v8":"4.2.77.18","uv":"1.4.2","zlib":"1.2.8","openssl":"1.0.2a","modules":"42"},
....
]
```

I'm asking because for node-pre-gyp I need to get this number right. I can start parsing the `node_versions.h` header for all formal releases if `release/index.js` is not intended to be stable. But I'd sure love it to be stable. Let me know if there is something I can help with.

**Comment by rvagg** [comment]
grr, a persistent bug that I apparently haven't squashed! something to do with the way dist-indexer caches values and how it handles lookup failures, I _think_. I'll remove the cache and run it again but that's not going to solve the bug obviously.

**Comment by springmeyer** [comment]
Thanks, that fixed it! Also noticing it fixed the 1.1.x -> 1.8.x versions which were reporting 42 but should be reporting 43 (now fixed).
